### PR TITLE
netman: update module description

### PR DESCRIPTION
**Pull request by mrpollo**
clarifies a few key aspects of the module for our docs here https://docs.px4.io/main/en/modules/modules_system.html#netman

Hey @davids5 it would be good to get your eyes on this one I want to make sure I captured the details correctly, I think the added info will help users.

### CODE REVIEW

File: src/systemcmds/netman/netman.cpp [hunk 1]
```diff
@@ -405,12 +405,21 @@ static void usage(const char *reason)
...
+  
+  ### update
+  On subsequent boots, the `update` option will check for the existence of
+  `net.cfg` in the root of the SD Card. It will saves the network settings
   from `net.cfg` in non-volatile memory, delete the file and reboot the system.
 
-  The `save` option will `net.cfg` on the SD Card. Use this to edit the settings.
-  The  `show` option will display the network settings  to the console.
+  ### save
+  The `save` option will write settings to the `net.cfg` file on the SD Card filesystem.
```
Comment (davids5) [inline src/systemcmds/netman/netman.cpp:415]:
```suggestion
  The `save` option will save settings, form non-volatile memory, to a file named `net.cfg` on the SD Card filesystem.
```

File: src/systemcmds/netman/netman.cpp [hunk 2]
```diff
@@ -405,12 +405,21 @@ static void usage(const char *reason)
   memory. On boot the `update` option will be run. If a network configuration
   does not exist. The default setting will be saved in non-volatile and the
   system rebooted.
-  On Subsequent boots, the `update` option will check for the existence of
-  `net.cfg` in the root of the SD Card.  It will saves the network settings
+  
+  ### update
+  On subsequent boots, the `update` option will check for the existence of
```
Comment (davids5) [inline src/systemcmds/netman/netman.cpp:410]:
```suggestion
  `netman update` is run automatically by the startup script.  the `update` option will check for the existence of
```

**Review by davids5 — CHANGES_REQUESTED** [review]
see inline

File: src/systemcmds/netman/netman.cpp (hunk 2 above)
Comment (mrpollo) [reply src/systemcmds/netman/netman.cpp:410]: Hey @davids5, which startup script are you referring too here?

**Comment by mrpollo** [comment]
Hey @davids5, which startup script is responsible for running netman update on boot?

File: src/systemcmds/netman/netman.cpp [hunk 3]
```diff
@@ -405,12 +405,22 @@ static void usage(const char *reason)
...
+
+  ### save
+  The `save` option will save settings from non-volatile memory to a file named
+  `net.cfg` on the SD Card filesystem for editing. Use this to edit the settings. 
+  Save does not immediately apply the network settings; it depends on the user
+  rebooting; Where the `update` command is run by the start-up script and also saves the settings.
+  
+    
+  ### show
+  The `show` option will display the network settings in `net.cfg` to the console.
```
Comment (hamishwillee) [inline src/systemcmds/netman/netman.cpp:423]: 1. The headings should be part of description.
2. Is this a particular startup script, such as extras.txt, a board config, an airframe config, or just "some part of the startup"?
   If the later, we might change to from "the startup script" to as shown below

```suggestion
  #### update
  
  `netman update` is run automatically by [a startup script](../concept/system_startup.md#system-startup).
  When run, the `update` option will check for the existence of `net.cfg` in the root of the SD Card.
  It then saves the network settings from `net.cfg` in non-volatile memory, 
  deletes the file and reboots the system.

  #### save
  
  The `save` option will save settings from non-volatile memory to a file named
  `net.cfg` on the SD Card filesystem for editing. Use this to edit the settings. 
  Save does not immediately apply the network settings; the user must reboot the flight stack.
  By contrast, the `update` command is run by the start-up script and also saves the settings.
  
  #### show
  
  The `show` option will display the network settings in `net.cfg` to the console.
```

File: src/systemcmds/netman/netman.cpp [hunk 4]
```diff
@@ -405,12 +405,22 @@ static void usage(const char *reason)
...
+  `netman update` is run automatically by the startup script. When run,
+  the `update` option will check for the existence of `net.cfg` in the
+  root of the SD Card. It will saves the network settings from `net.cfg`
+  in non-volatile memory, delete the file and reboot the system.
+
+  ### save
+  The `save` option will save settings from non-volatile memory to a file named
+  `net.cfg` on the SD Card filesystem for editing. Use this to edit the settings. 
+  Save does not immediately apply the network settings; it depends on the user
+  rebooting; Where the `update` command is run by the start-up script and also saves the settings.
```
Comment (davids5) [inline src/systemcmds/netman/netman.cpp:419]:
```suggestion
  rebooting; Where the `update` command is run by the start-up script.
```

File: src/systemcmds/netman/netman.cpp (hunk 3 above)
Comment (davids5) [reply src/systemcmds/netman/netman.cpp:423]: change `also saves the settings.` to `and commits the settings to non-volatile memory, reboots the flight controller which will then use the new settings.`

Comment (hamishwillee) [reply src/systemcmds/netman/netman.cpp:423]: Thanks @davids5 done, and I'll commit.

**Comment by hamishwillee** [comment]
>  which startup script is responsible for running netman update on boot?

@mrpollo Thanks for updating all this doc. I answered this as "some part of the startup" by providing a link to the start up docs.

@junwoo091400 Thanks for tagging some of these with documentation. Historically no one has bothered with that, and so I have not been monitoring. I will do so now that I know you are watching that. I don't have time to look at everything, but very happy to look at things that are relevant.

**Comment by davids5** [comment]
> Hey @davids5, which startup script is responsible for running netman update on boot?

it is in rcS line 167

**Review by davids5 — APPROVED** [review]
LGTM

**Pull request merged**